application/views/admin/expressions/test/survey_logic_file.php

```diff
@@ -32,4 +32,27 @@
             </div>
         </div>
     </div>
-</div>
\ No newline at end of file
+</div>
+
+<style>
+    /**
+     * This whole style block is a workaround for https://bugs.limesurvey.org/view.php?id=18250
+     */
+    .main-content-container {
+        overflow-x: visible;
```

Comment:
Did we really need to `overflow-x: hidden; ` in all other part ? Can not be removed totally ?

Reply:
Not sure what you mean... Still, this is a workaround for doing a minimum update and avoiding a complete redo.

Reply:
I mean : in general overflow : hidden is a bad solution and must be avoid ;)